dev.sh

```diff
@@ -1,4 +1,45 @@
 #!/bin/bash
 
-./db.sh
-pnpm run dev
\ No newline at end of file
+
+if [ "$EUID" -ne 0 ];
+  then echo "Please run as root. This is due to some overriding in certificates and the hosts file"
+  exit
+fi
+
+
+trap '' INT
+
+if [ ! -x "$(command -v mkcert)" ];
+then
+    curl -JLO "https://github.com/FiloSottile/mkcert/releases/download/v1.4.4/mkcert-v1.4.4-linux-amd64"
+    chmod +x mkcert-v*-linux-amd64
+    cp mkcert-v*-linux-amd64 /usr/local/bin/mkcert
+    rm mkcert-v*-linux-amd64
+fi
+if [[ ! -f localhost.cert ]]
+then
+    mkcert -install
```

Comment:
Again, this shouldn't be done automatically. It should be up to the dev to do this after installing mkcert themselves, especially since it's a one-time operation.